article-label-to-content-tag/README.md

```diff
@@ -0,0 +1,29 @@
+# Zendesk Gather example API client for user contribution stats
+
+This is a simple utility built in Python that converts an Article Label into a public-facing Content Tags
+
+## Requirements
+
+This tool requires Python 3.x and the [requests](https://pypi.org/project/requests/) library installed.
+
+If you already have `pip`, simply install like this:
+
+```
+pip install requests
+```
+
+## Usage
+
+This is a command line tool. Invoke it like this:
+
+```
+python main.py -d <subdomain> -u <username> -t <token> -o <output_file> --from=<from> --to=<to>
```

Comment:
```suggestion
python main.py -d {subdomain} -u {email_address} -t {api_token}  --label {article_label} --tag {content_tag}
```

I couldn't get the `-o` flag to work. I also updated some of these so they're more aligned with the rest of our docs.

Reply:
Doh this was a bad copy-paste from a previous python script I had